code/modules/economy/account.dm

```diff
@@ -232,4 +252,12 @@
 	exclusive_budget_pool = NON_STATION_BUDGET_BASE
 	..()
 
+// all golems will share this account on their cards. the unknown RD wasn't surely a rich who can make a bank account for every golem.
+/datum/bank_account/department/mining_golem
+	account_holder = ACCOUNT_GOLEM_NAME
+	department_id = ACCOUNT_GOLEM_ID
+	department_bitflag = NONE
+	show_budget_information = TRUE // It's shared on all golem cards
+	exclusive_budget_pool = 13 // oh no, someone used it! damn communism!
```

Comment:
shouldn't this be a "nonstation" account or whatever

Reply:
It's already nonstation.
`nonstation_account` var is `TRUE` at its parent, and station accounts are set to `FALSE` for it.